### DISCUSSION TITLE
LLAMA_3.1 finetune?

### DISCUSSION THREAD

**Discussion by SicariusSicariiStuff** [axolotl-ai-cloud/axolotl#1786] [Q&A]
Hi,
I read there are still some bugs with the new llama 3.1 (8b, i can't sell my house to train a 405b) because of its rope and other context window issues.

Can it be trained successfully in the current version of axolotl, or do I need to change something specific for it to work?

**Comment by tridungduong-unsw** [comment]
Same issue, I've tried to fine-tune LLAMA 3.1, but it doesn't work with logs as below:

`ValueError: rope_scaling must be a dictionary with two fields, type and factor, got {'factor': 8.0, 'low_freq_factor': 1.0, 'high_freq_factor': 4.0, 'original_max_position_embeddings': 8192, 'rope_type': 'llama3'}`

**Comment by leemgs** [comment]
From my experience, you need to update the existing Transformers  to fine-tune Llama 3.1.

**Comment by winglian** [comment]
latest transformers should work
